fastpath/fastpath/core.py

```diff
@@ -1662,7 +1662,17 @@ def process_measurement(msm_tup, buffer_writes=False) -> None:
         engine_name = g_or(annot, "engine_name", "")
         engine_version = g_or(annot, "engine_version", "")
         blocking_type = g(scores, "analysis", "blocking_type", default="")
+        ooni_run_link_id_str = g_or(annot, "ooni_run_link_id", "")
+        if ooni_run_link_id_str:
+            try:
+                ooni_run_link_id = int(ooni_run_link_id_str)
+            except ValueError:
+                ooni_run_link_id = None
+        else:
+            ooni_run_link_id = None
```

Comment:
Isn't it a bit more clear to rewrite this as:

```suggestion
        ooni_run_link_id_str = g_or(annot, "ooni_run_link_id", None)
        try:
            ooni_run_link_id = int(ooni_run_link_id_str)
        except (TypeError, ValueError):
            ooni_run_link_id = None
```

?

Reply:
It would be shorter but it would raise and catch an exception around int() even for the expected value `""`.

Reply:
It would not raise the exception, since `int("")` throws a `ValueError` which is caught by the handler.

It's equivalent to the previous code.